src/frontend/query_commands.ml

```diff
@@ -512,6 +512,29 @@ let dispatch pipeline (type a) : a Query_protocol.t -> a =
     | Some res -> `Found res 
     | None -> `No_documentation)
 
+  | Expand_node pos -> 
+    let pos = Mpipeline.get_lexing_pos pipeline pos in
+    let typer = Mpipeline.typer_result pipeline in 
+    let node = Mtyper.node_at typer pos in
+    let check_ppx (node: Env.t * Mbrowse.node) : bool =
```

Comment:
Is the type anotation really mandatory? 
`let check_ppx node = ...` seems sufficient no?